seirmo/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#"""
```

Comment:
We forgot to do this with @FarmHJ PR.

Could you please register the FowardModel class by importing it like this:
```
from .models import (
   ForwardModel
)

```